src/components/landing/pages/history/common/pin-button.scss

```
@@ -1,12 +1,15 @@
 .history-pin {
-  opacity: 0.2;
-  transition: opacity 0.2s ease-in-out, color 0.2s ease-in-out;
 
-  &:hover {
+  .fa {
+    opacity: 0.2;
+    transition: opacity 0.2s ease-in-out, color 0.2s ease-in-out;
+  }
+
+  &:hover .fa{
     opacity: 1;
   }
 
-  &.active {
+  &.pinned .fa{
```

Same as above (missing space)
```suggestion
  &.pinned .fa {
```